store/fish.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
+import { baseMutations } from '@/utils/store_actions';
+
+const route = 'fish';
+
+export const state = () => ({
+  coordinates: [],
+  samples: [],
+  temps: [],
+  stations: [],
+  species: [],
+  info: {}
+});
+
+export const mutations = { ...baseMutations };
+
+export const actions = {
+  async fetchCoordinates({ commit, dispatch }) {
+    const coords = await this.$axios.$get(`/${route}/coordinates`);
+    const stations = coords.map(({ station_name }) => station_name).sort();
+    commit('mutate', { property: 'stations', with: stations });
+    commit('mutate', { property: 'coordinates', with: coords });
+    dispatch(
+      'updateColorMap',
+      { ids: coords.map((v) => v.station_name), unique: false },
+      { root: true }
+    );
+  },
+  async fetchSamples({ commit }) {
+    const samples = await this.$axios.$get(`/${route}/species`);
+    let species = samples.map(({title}) => title).sort()
+    species = species.filter((v, i, a) => a.indexOf(v) === i);
+    commit('mutate', { property: 'species', with: species })
+    const dataParsed = samples.map(d => {
+      if (d.species.includes('crab')){
```

Comment:
```suggestion
      if (d.species.toLowerCase().includes('crab')){
```

Suggestion to match the rest of this if branching. Should `d.species.toLowerCase()` be saved in a temp variable?